lib/options.ts

```diff
@@ -17,7 +17,8 @@ var knownOpts:any = {
 		"keyStorePath": String,
 		"keyStorePassword": String,
 		"keyStoreAlias": String,
-		"keyStoreAliasPassword": String
+		"keyStoreAliasPassword": String,
+		"debug-brk": Boolean
```

Comment:
You should add the option as debug-brk and debugBrk here. The reason is that we are using yargs to get command line arguments and it is working this way: 
If you pass value with dash, yargs adds it to yargs.argv in two ways: with dash and without dash, replacing first symbol after it with its toUpper equivalent.
As we are validating all passed arguments, when you try using debug-brk, yargs will pass two values to our validation logic  - debug-brk and debugBrk. Just add it in both ways, so the validation will pass.